src/routes/SafeAppLandingPage/SafeAppLandingPage.test.tsx

```diff
@@ -0,0 +1,208 @@
+import * as safeAppsGatewaySDK from '@gnosis.pm/safe-react-gateway-sdk'
+
+import SafeAppLandingPage from './SafeAppLandingPage'
+import { render, screen, waitFor, waitForElementToBeRemoved } from 'src/utils/test-utils'
+import { history, OPEN_SAFE_ROUTE, WELCOME_ROUTE } from 'src/routes/routes'
+import * as appUtils from 'src/routes/safe/components/Apps/utils'
+import { FETCH_STATUS } from 'src/utils/requests'
+import { saveToStorage } from 'src/utils/storage'
+
+const SAFE_APP_URL_FROM_CONFIG_SERVICE = 'https://safe-app.gnosis-safe.io/test-safe-app-from-config-service'
+const SAFE_APP_URL_FROM_MANIFEST = 'https://safe-app.gnosis-safe.io/test-safe-app-from-manifest'
+const SAFE_APP_CHAIN_ID = '4'
+
+describe('<SafeAppLandingPage>', () => {
+  beforeEach(() => {
+    // mocking safe apps list from the Config Service endpoint
+    jest.spyOn(safeAppsGatewaySDK, 'getSafeApps').mockReturnValue(
```

Comment:
No need to change this but there is also [`mockResolvedValue()`](https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue). You won't need to specify `Promise.resolve()`.